extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerUtil.java

```diff
@@ -86,7 +89,7 @@ static PolicyEnforcer createPolicyEnforcer(OidcTenantConfig oidcConfig,
                 .credentials(adapterConfig.getCredentials())
                 .bearerOnly(adapterConfig.isBearerOnly())
                 .enforcerConfig(enforcerConfig)
-                .httpClient(new HttpClientBuilder().build(adapterConfig))
+                .httpClient(new HttpClientBuilder().sslContext(tlsConfigSupport.getSslContext()).build(adapterConfig))
```

Comment:
What kind of HTTP client is it? 
If it's vert.x based, it might be better to use the dedicated Vert.x options. They support a lot more parameters than the SSL context.

Reply:
It has to be `org.apache.http.client.HttpClient` because of what Keycloak PolicyEnforcer accepts, it's out of our hands. Therefore it's either SSL context or manually mapping from config https://github.com/quarkusio/quarkus/blob/0a21b51e652e523cadf7866f5963db169d72e66f/extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/HttpClientBuilder.java#L286

IMO we barely hear about KC AuthZ so I wonder how much is it used, but we could ask Pedro for Vert.X based client.

Reply:
ah ok! Thanks!